src/main/java/ch/hevs/cloudio/endpoint/JobsParameter.java

```diff
@@ -0,0 +1,22 @@
+package ch.hevs.cloudio.endpoint;
+
+public class JobsParameter {
+    private String jobURI;
+    private boolean getOuput;
```

Comment:
sendOutput might be a more suitable name

Reply:
make sense, sendOutput it will be